From 2ef733b9b65f77b43786c9bd08745416e23521ea Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <jan.steffens@gmail.com>
Date: Tue, 16 Aug 2016 03:14:15 +0200
Subject: [PATCH] user dbus.socket: Remove [Install] Also=dbus.service

When enabling dbus.socket, the cycle between dbus.socket and
dbus.service is broken by ignoring the Also=dbus.socket. Since the
install section of dbus.service is otherwise empty, systemctl then
complains that dbus.service is a static unit.

Break the cycle ourselves by removing Also=dbus.service. The intended
behavior of enabling the socket when enabling the service remains.
---
 bus/systemd-user/dbus.socket.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/bus/systemd-user/dbus.socket.in b/bus/systemd-user/dbus.socket.in
index 4793d1d4ff1dc13e..3b56b3e7f6603024 100644
--- a/bus/systemd-user/dbus.socket.in
+++ b/bus/systemd-user/dbus.socket.in
@@ -7,4 +7,3 @@ ExecStartPost=-@SYSTEMCTL@ --user set-environment DBUS_SESSION_BUS_ADDRESS=unix:
 
 [Install]
 WantedBy=sockets.target
-Also=dbus.service
-- 
2.9.3

